app/editor/src/hooks/api-editor/interfaces/ITimeTrackingModel.ts

```diff
@@ -5,4 +5,5 @@ export interface ITimeTrackingModel {
   activity: string;
   userName?: string;
   createdOn?: any;
+  id?: number;
```

Comment:
Is `id` truly undefinable?  Or is it mostly likely `0` for a new row?

Reply:
In some areas of the app (when passing the parsed data to the pop up) the id is undefined at that time, could adjust this if needed

Reply:
The model should represent what the API will return.  Does the API ever return null?